code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -367,7 +367,7 @@ to_chat will check for valid clients itself already so no need to double check f
 	if(!stored_larva) // no larva to deal with
 		return ..()
 
-	stored_larva = round(stored_larva * QUEEN_DEATH_LARVA_MULTIPLIER(Q)
+	stored_larva = round(stored_larva * QUEEN_DEATH_LARVA_MULTIPLIER(Q), 1)

```

Comment:
It already rounds to the nearest integer by default, no need to explicate the latter arg.

Reply:
Personally don't agree as I feel explicit is better than implicit, that said, i made the change regardless.